.azuredevops/pipelines/acr-image-retag.yaml

```diff
@@ -4,10 +4,21 @@ name: $(Build.SourceBranchName)-$(Date:yyyyMMdd)_$(Rev:r)
 trigger: none
 pr: none
 
+resources:
+  repositories:
+    - repository: dtos-devops-templates
+      type: github
+      name: NHSDigital/dtos-devops-templates
+      ref: fa58dc978491f04e1efab73cbf8e2228a351bf81
```

Comment:
set this to main

Reply:
Will leave as is. This is a commit in main. We have already agreed this strategy, since it reduces fragility in future.